src/sentry/static/sentry/app/views/onboarding/configure.jsx

```diff
@@ -88,7 +88,11 @@ const Configure = createReactClass({
           <h2 style={{marginBottom: 30}}>
             {t('Configure your application')}
             {!hasSentRealEvent && (
-              <CreateSampleEvent params={this.props.params} source="header" />
+              <div className="pull-right">
```

Comment:
Can we style the parent wrapper instead?

Reply:
I didn't want to fuss with this onboarding page since it will go away soon.